runtime/plugins/org.jboss.tools.runtime.ui/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,8 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.core.resources,
  org.eclipse.ui.navigator;bundle-version="3.5.100",
  org.jboss.tools.foundation.core;bundle-version="1.1.0",
- org.jboss.tools.foundation.ui;bundle-version="1.1.0"

```

Comment:
I don't believe a dependency on tools.common is appropriate here. I think you probably forgot to remove it.  We are trying to do more work in foundation and less in common. This patch, itself, puts all of the new code in foundation.